app.js

```diff
@@ -7,56 +7,165 @@ console.log("var = animals asks the user if I like animals");
-}
-else if (juice.trim().toLowerCase() == 'no' || juice.trim().toLowerCase() == 'n' ){
-  alert('I lived in Italy and Afghanistan!');
-}
-else{
-  alert("Your response wasn't yes or no!");
-}
+
+  if (travel == null) {
+    alert('Canceled');
+  }else if (travel.trim().toLowerCase() == 'yes' || travel.trim().toLowerCase() == 'y' ){
+    alert('You are correct!');
+  }else if (travel.trim().toLowerCase() == 'no' || travel.trim().toLowerCase() == 'n' ){
+    alert('I lived in Italy and Afghanistan!');
+  }else{
+    alert("Your response wasn't yes or no!");
+  }
 
 var animals = prompt("Am I an animal lover? (answer yes or no)");
-if (animals.trim().toLowerCase() == 'yes' || animals.trim().toLowerCase() == 'y' ){
+
+if (animals == null) {
+  alert('Canceled');
+}else if (animals.trim().toLowerCase() == 'yes' || animals.trim().toLowerCase() == 'y' ){
   alert('You are correct!');
-}
-else if (animals.trim().toLowerCase() == 'no' || animals.trim().toLowerCase() == 'n' ){
+}else if (animals.trim().toLowerCase() == 'no' || animals.trim().toLowerCase() == 'n' ){
   alert('But I LOVE animals!');
-}
-else{
+}else{
   alert("Your response wasn't yes or no!");
 }
 
 var people = prompt('Do I like people? (answer yes or no)');
-if (people.trim().toLowerCase() == 'yes' || people.trim().toLowerCase() == 'y' ){
+
+if (people == null) {
+  alert('Canceled');
+}else if (people.trim().toLowerCase() == 'yes' || people.trim().toLowerCase() == 'y' ){
   alert('You are correct!');
-}
-else if (people.trim().toLowerCase() == 'no' || people.trim().toLowerCase() == 'n' ){
+}else if (people.trim().toLowerCase() == 'no' || people.trim().toLowerCase() == 'n' ){
   alert('What kind of person do you think I am!?');
-}
-else{
+}else{
   alert("Your response wasn't yes or no!");
 }
+
+var number = 7;
+var done = true;
+var rightNum;
+for (var num = 4; num > 0 && done; num--){
+  var rightNum = prompt('What is my favorite number?');
+  rightNum = parseInt(rightNum);
+
+  if(rightNum === number){
+    alert('Wow! You guessed the correct number!!');
+    done = false;
+
+  }else if(num === 1){
+      alert('You ran out of guesses!');
+
+  }else if(rightNum > number){
+    alert('You\'re too high!');
+
+  }else{
+    alert('You\'re too low!');
+  }
+}
+
+var countries = ['afghanistan', 'italy', 'germany']
+var rightCountry;
+var done2 = true;
+for (var place = 0; place < 6 && done2; place++){
+  var rightCountry = prompt('Can you name a country that I lived in?');
+
+  if (rightCountry.toLowerCase() === countries[0] || rightCountry.toLowerCase() === countries[1] || rightCountry.toLowerCase() === countries[2]){
+    alert('Wow! You guessed a country I lived in!');
+    done2 = false;
+
+  }else if(place === 5){
+    alert('You ran out of guesses!!');
+
+  }else{
+    alert('I never lived there!');
+  }
+
+}
+
+var tally = 0;
+if(rightCountry === 'italy'){
+     tally++;
+}
+if(rightCountry === 'germany'){
+  tally++;
+}if(rightCountry === 'afghanistan'){
+  tally++;
```

Comment:
you can use your logical operators to collapse these into one if statement. Should make it more readable, too!